.gitignore

```diff
@@ -8,6 +8,7 @@
 *.rbc
 *.sassc
 **.orig
+*.lock
```

Comment:
What are you trying to avoid tracking with this entry? We definitely want to track our `Gemfile.lock` and `yarn.loc`k files.

Reply:
I didn't intend to push this.